security.rst

```diff
@@ -1767,9 +1767,12 @@ You can log in a user programmatically using the ``login()`` method of the
             // you can also log in on a different firewall...
             $security->login($user, 'form_login', 'other_firewall');
 
-            // ...and add badges
+            // ... add badges...
             $security->login($user, 'form_login', 'other_firewall', [(new RememberMeBadge())->enable()]);
 
+            // ... and also add passport attributes
+            $security->login($user, 'form_login', 'other_firewall', [(new RememberMeBadge())->enable()], ['arbitrary_key' => 'some data']);
```

Comment:
Let's use some real values here for example. 

Referer maybe?

Reply:
I tried to update according to your comment. Is it what you had in mind?

Reply:
Perfect, thanks